rust/processor/src/db/common/models/default_models/parquet_write_set_changes.rs

```diff
@@ -209,21 +228,32 @@ impl WriteSetChange {
         block_height: i64,
         timestamp: chrono::NaiveDateTime,
     ) -> (Vec<Self>, Vec<WriteSetChangeDetail>) {
-        write_set_changes
+        let results: Vec<(Self, WriteSetChangeDetail)> = write_set_changes
             .iter()
             .enumerate()
-            .map(|(write_set_change_index, write_set_change)| {
-                Self::from_write_set_change(
+            .filter_map(|(write_set_change_index, write_set_change)| {
+                match Self::from_write_set_change(
                     write_set_change,
                     write_set_change_index as i64,
                     txn_version,
                     block_height,
                     timestamp,
-                )
+                ) {
+                    Ok(Some((change, detail))) => Some((change, detail)),
+                    Ok(None) => None,
+                    Err(e) => {
+                        tracing::error!(
+                            "Failed to convert write set change: {:?} with error: {:?}",
+                            write_set_change,
+                            e
+                        );
```

Comment:
Is this safe? I feel like normally in the processors we prefer to fail rather than log an error and move on. This way the processor can "succeed"  but drop write set changes. If there is a good reason why this is okay here, could you leave a comment?

Reply:
changed to fail rather than log an error and move on.